### PR TITLE
Update naming format for k3s and rke1 clusters modules + add batch_scale scripts

### PR DESCRIPTION
**Pull request by git-ival**

### CODE REVIEW

**Review by floatingman — APPROVED** [review]
Ran a successful small scale test using both k3s and rk1 clusters and the batch scaling script

File: clusters-rke1/README.md [hunk 1]
```diff
@@ -54,7 +54,7 @@ To cleanup, there is a `destroy_clusters.sh`, this script will perform a `terraf
 |------|------|
 | [null_resource.nt_dependency](https://registry.terraform.io/providers/hashicorp/null/latest/docs/resources/resource) | resource |
 | [rancher2_cluster.rke1](https://registry.terraform.io/providers/rancher/rancher2/latest/docs/resources/cluster) | resource |
-| [rancher2_node_pool.aws_np](https://registry.terraform.io/providers/rancher/rancher2/latest/docs/resources/node_pool) | resource |
+| [rancher2_node_pool.np](https://registry.terraform.io/providers/rancher/rancher2/latest/docs/resources/node_pool) | resource |
```
Comment (izaac) [inline clusters-rke1/README.md:57]: Just curious on the README, is this automatically generated from the variables file?

File: clusters-rke1/batch_scale.sh [hunk 2]
```diff
@@ -0,0 +1,84 @@
...
+KUBE_CONFIG=${1}
+BATCH_NUM_NODES=${2:-50}
+TARGET_NUM_DOWNSTREAMS=${3:-150}
+
+export KUBECONFIG="${KUBE_CONFIG}"
+
+function get_heap_logs() {
+    for pod in $(kubectl -n cattle-system get pods --no-headers -l app=rancher | cut -d ' ' -f1); do
+        echo getting profile for "${pod}"
+        kubectl -n cattle-system exec "${pod}" -- curl -s http://localhost:6060/debug/pprof/heap -o profile.log
```
Comment (izaac) [inline clusters-rke1/batch_scale.sh:12]: Is port `6060` a standard meaning does it change between cluster types?

File: clusters-rke1/main.tf [hunk 3]
```diff
@@ -53,6 +53,7 @@ locals {
   cloud_cred_name          = "${local.rancher_subdomain}-aws-cloud-cred"
   node_template_name       = "${local.rancher_subdomain}-aws-nt"
   nt_depends_on            = var.create_node_reqs ? module.shared_node_template[0] : data.rancher2_node_template.existing_nt[0]
+  cluster_name             = "${local.rancher_subdomain}-${var.cluster_name}-${terraform.workspace}"
```
Comment (izaac) [inline clusters-rke1/main.tf:56]: do we need something like a UUID to prevent name collisions or is it a very unlikely scenario ?

**Review by izaac — COMMENTED** [review]
Just a couple comments.

File: clusters-rke1/batch_scale.sh (hunk 2 above)
Comment (git-ival) [reply clusters-rke1/batch_scale.sh:12]: 6060 is specific to whatever http server port the application is using to ListenAndServe with, [pprof](https://pkg.go.dev/net/http/pprof) can then profile on that port. In this case [Rancher](https://github.com/rancher/rancher/blob/127d5b63a7ce1ca26aea9266926997f60bfad4f4/main.go#L25) is using 6060.

File: clusters-rke1/main.tf (hunk 3 above)
Comment (izaac) [reply clusters-rke1/main.tf:56]: I get it, sounds good then.

File: clusters-rke1/README.md (hunk 1 above)
Comment (izaac) [reply clusters-rke1/README.md:57]: That's neat!

**Review by izaac — APPROVED** [review]
🚀 LGTM

**Pull request merged**